README.md

```diff
@@ -26,3 +26,18 @@ SFTP_PORT                                        | The port over which to connec
 SFTP_USERNAME                                    | The username of the SFTP server credentials.                                                                  | 
 SFTP_PASSWORD                                    | The password of the SFTP server credentials.                                                                  |
 SFTP_FILE_PATH                                   | The file path, relative to the root of the SFTP server, to which to upload CSV files.                         | 'uploadPath' (will result is CV's uploaded to directory: ~/uploadPath)
+
+### Test Environment Variables
+Environment variables required to execute integration tests for this lambda:
+
+Name                | Description                                                                           | Examples
+--------------------| --------------------------------------------------------------------------------------|--------------------------------------------------------------------------
+ENV_REGION_AWS      | The AWS region code for the lambda.                                                   | 'eu-west-2'         
```

Comment:
Backticks would give better formatting for the examples
```suggestion
ENV_REGION_AWS      | The AWS region code for the lambda.                                                   | `eu-west-2`
```

Reply:
Updated all examples to use backticks